internal/brokers/testdata/TestIsAuthenticated/golden/error_when_broker_returns_invalid_userinfo

```diff
@@ -0,0 +1,4 @@
+FIRST CALL:
+	access: 
+	data: 
+	err: invalid user information: message is not JSON formatted: json: cannot unmarshal string into Go value of type struct { cache.UserInfo; UUID string; UGID string; Groups []struct { Name string; UGID string } }
```

Comment:
yeah, see, but looking at the golden files, we can see the error is missing some information, like the raw data.

Reply:
{ See my comment about leaks }